nanome/_internal/_ui/_deprecated/_button_deprecated.py

```diff
@@ -0,0 +1,216 @@
+import nanome
```

Comment:
put all the deprecated code here so it wouldn't take up a ton of space. It is just added to the button class using python magic.